modin/engines/ray/pandas_on_ray/frame/modin_aqp.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import ray
+import os
+import time
+import threading
+import warnings
+
+progress_bars = {}
+bar_lock = threading.Lock()
+
+
+def call_progress_bar(result_parts, line_no):
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        try:
+            from tqdm.autonotebook import tqdm as tqdm_notebook
+        except ImportError:
+            raise ImportError("Please pip install tqdm to use the progress bar")
+        from IPython import get_ipython
+
+    try:
+        cell_no = get_ipython().execution_count
+    # This happens if we are not in ipython or jupyter.
+    # No progress bar is supported in that case.
+    except AttributeError:
+        return
+    pbar_id = str(cell_no) + "-" + str(line_no)
+    futures = [x.oid for row in result_parts for x in row]
+    bar_format = (
+        "{l_bar}{bar}{r_bar}"
+        if "DEBUG_PROGRESS_BAR" in os.environ
```

Comment:
Where is `DEBUG_PROGRESS_BAR` defined? It should probably be `MODIN_PROGRESS_BAR` that is introduced in `modin.config`.

Reply:
see https://github.com/modin-project/modin/pull/2677#discussion_r588043503